### ISSUE TITLE
Copy rules from flake8-raise

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1184]
I found this lovely plugin: https://github.com/jdufresne/flake8-raise

It has three rule that I absolutely love to have in `0.15`:

- [ ] `R100 raise in except handler without from`
- [ ] `R101 use bare raise in except handler`

Declined:
- [x] `R102 unnecessary parentheses on raised exception`

First step is to check that all three violations are not already supported by our linter.
The second step is to create new issues for violations that should be created.

**Comment by sobolevn** [comment]
Nope, `R102` is excluded from the list. We cannot assume type of the call.

**Comment by uhbif19** [comment]
@sobolevn Can make this. Would be using the new CST visitor ok for this?

**Comment by sobolevn** [comment]
@uhbif19 sorry for the misinformation. `R102` is out of scope. Only two rules are required.
You are assigned! Thanks a lot for your help!

**Comment by uhbif19** [comment]
@sobolevn 

1. Looks like `R101` is covered by `WPS329`.

https://wemake-python-stylegui.de/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.UselessExceptCaseViolation

2. My test case and doc for `WPS329` shows that R100 is not covered.

**Comment by sobolevn** [comment]
Yes, @uhbif19! Let's move to #1407 

Thanks!

**Comment by uhbif19** [comment]
@sobolevn Turns out, I was wrong. `R101` is covered only in the case, then `raise` is the only statement in `expect` block.

That case generates violation:

```
except ZeroDivisionError:
    raise
```

And this does not:

```
except ZeroDivisionError:
    my_print('something')
    raise
```

**Comment by sobolevn** [comment]
And it's fine 🙂

**Comment by sobolevn** [comment]
See #1407
